### PR TITLE
Qt: Add "Move Support" column to the gamelist

### PR DESCRIPTION
**Pull request by Megamouse**
Also fixes some sorting issues and hides path and firmware by default

### CODE REVIEW

**Comment by Megamouse** [comment]
it is expected. you need to either delete the currentsettings.ini inside the guiconfigs folder or remove the gamelist state entry in that file.

then it should work.

**Comment by legend800** [comment]
Didn't work after deleting that file. Same behavior - extra 'show icons', everything offset.

**Comment by Megamouse** [comment]
I found the bug.
It will be handled in my newest pull request: https://github.com/RPCS3/rpcs3/pull/4604

I replaced the old naming convention in the gui settings for better future consistency.
It will reset the current column settings to default though.

File: rpcs3/rpcs3qt/game_compatibility.cpp [hunk 1]
```diff
@@ -162,7 +162,7 @@ void game_compatibility::RequestCompatibility(bool online)
 	});
 
 	// Handle response according to its contents
-	connect(network_reply, &QNetworkReply::finished, [=]()
+	connect(network_reply, &QNetworkReply::finished, [&]()
```
Comment (Kravickas) [inline rpcs3/rpcs3qt/game_compatibility.cpp]: This is causing segfault

**Pull request merged**